magicbot.js

```diff
@@ -19,7 +19,7 @@ module.exports = function(req, res, next){
         payload.text = result.statusCode == 404 ? "Sorry, could not find a card by that name."
                                                 : "*Card name:* " + parsedBody.name +
                                                   "\n*Text:* " + parsedBody.text +
-                                                  "\n*Image:* " + parsedBody.editions[0].image_url;
+                                                  "\n*Image:* " + parsedBody.editions.reverse()[0].image_url;

```

Comment:
Keep in mind that `Array#reverse` mutates the array. We could use a pure expression instead:

```javascript
parsedBody.editions[parsedBody.editions.length - 1].image_url
```